### PR TITLE
[Chore #150397528] Configure Eslint rules

### PR DESCRIPTION
**Pull request by Dannytebj**
Configured Eslint rules

### CODE REVIEW

**Comment by THEozmic** [comment]
I see that you also made changes to package.json, why is this?

**Comment by Dannytebj** [comment]
So after configuring eslint rules, I ran`npm install eslint --save-dev` , and I guess that made changes to package.json!

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+"root": true,
```
Comment (mradeybee) [inline .eslintrc:2]: We might need to add more rules. Were you given any rules to use?

Comment (Dannytebj) [reply .eslintrc:2]: Okay, Yeah, these were the rules given in pivotal tracker

**Review by mradeybee — APPROVED** [review]
GTM 👍

**Pull request merged**